packages/noodl-editor/src/editor/src/views/panels/VersionControlPanel/components/DiffList.tsx

```diff
@@ -61,7 +61,7 @@ export function DiffList({ diff, fileChanges, componentDiffTitle, actions, commi
   const [imageDiff, setImageDiff] = useState<IImageDiff>(null);
 
   const components = diff?.components ? getChangedComponents(diff.components) : [];
-  const files = (fileChanges || [])?.filter((f) => f.path !== 'project.json') || [];
+  const files = (fileChanges || [])?.filter((f) => !f.path.endsWith('project.json')) || [];
```

Comment:
This is just how Noodl is built, in the version control panel, it wont show files called "project.json".